### PR TITLE
test (ticdc): add cross database ddls cases

### LINKED ISSUE #10004: Add test cases to cover cross database DDLs.
### Is your feature request related to a problem?

Add some test cases to cover cross database DDLs like:
- Rename table db1.t1 to db2.t1.
- Create table db1.t2 like db2.t1.

### Describe the feature you'd like

as title.

### Describe alternatives you've considered

_No response_

### Teachability, Documentation, Adoption, Migration Strategy

_No response_

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #10004 

#### Tests 

 - Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: tests/integration_tests/ddl_manager/data/prepare.sql [hunk 1]
```diff
@@ -98,8 +98,37 @@ CREATE TABLE t10 (
...
+
+use `across_db_1`;
+
+CREATE TABLE t1 (
+                    id INT AUTO_INCREMENT PRIMARY KEY,
+                    val INT DEFAULT 0,
+                    col0 INT NOT NULL
+);
+
+RENAME TABLE `across_db_1.t1` TO `across_db_2.t1`;
```
Comment (nongfushanquan) [inline tests/integration_tests/ddl_manager/data/prepare.sql:119]: add a negative case, such as " rename table t1 to db2.t1"

Comment (asddongmen) [reply tests/integration_tests/ddl_manager/data/prepare.sql:119]: If it is added, the test will failed. I can add it in the comment.

**Review by nongfushanquan — APPROVED** [review]

**Review by CharlesCheung96 — APPROVED** [review]

**Pull request merged**